packages/apps/helpdesk/src/components/templates/TicketSearchBar/TicketSearchBar.js

```diff
@@ -43,14 +43,21 @@ const TicketSearchBar = ({
 
   const fetchTicketSearchBarAPI = useCallback(
     ({page = 0, searchValue}) => {
-      dispatch(
-        fetchTickets({
-          page: page,
-          searchValue: searchValue,
-          userId: team ? null : user.id,
-          userTeam: team ? user.activeTeam : null,
-        }),
-      );
+      team
+        ? dispatch(
+            fetchMyTeamTickets({
+              searchValue: searchValue,
+              userTeam: user.activeTeam,
+              page: page,
+            }),
+          )
```

Comment:
Caution, you are changing the call but not the list used in the searchBar.
I think it could be better to create two searchBar as functionnal use is completly different now.